instagram/src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import './App.css';
+import PostsPage from './components/PostContainer/PostsPage';
+import Authenticate from './components/Authenticate/Authenticate';
+
+class App extends Component {
+  constructor() {
+    super();
```

Comment:
Similarly here as well - `super()` calls the constructor of its parent class. This is required when you need to access some variables from the parent class.

Reply:
Thanks!